src/marten/db/query/sql/raw_predicate_node.cr

```diff
@@ -0,0 +1,70 @@
+require "./concerns/sanitizer"
+
+module Marten
+  module DB
+    module Query
+      module SQL
+        class RawPredicateNode < PredicateNode
+          include Sanitizer
+
+          getter statement
+          getter params
+
+          def initialize(
+            @statement : String,
+            @params = [] of ::DB::Any,
+            @children = [] of PredicateNode,
+            @connector = SQL::PredicateConnector::AND,
+            @negated = false
```

Comment:
I actually wonder if we should not avoid making it possible to initialize raw predicate nodes with `negated`. Because since we don't negate `RawNode` objects, we technically don't need to allow initializing `RawPredicateNode` with `negated: true`. Plus, we also don't use `negated` anywhere for raw predicate nodes. We could simply automatically initialize it to `true` (similarly to what we did for `predicates`). WDYT?

Reply:
Yes sounds like the right way to do!